### ISSUE TITLE
Any idea the feasibility of adding an exclude or ignore directory option?

### ISSUE THREAD

**Issue by groovenectar** [yurijmikhalevich/rclip#16]
E.g., like `rsync` has the `--exclude=PATTERN` that can be repeated...

I see this in [`main.py`](https://github.com/yurijmikhalevich/rclip/blob/main/rclip/main.py#L37):

```lang=python
EXCLUDE_DIRS = ['@eaDir', 'node_modules', '.git']
EXCLUDE_DIR_REGEX = re.compile(r'^.+\/(' + '|'.join(re.escape(dir) for dir in EXCLUDE_DIRS) + r')(\/.+)?$')
```

Do you have any interest in having additional exclude directories specified as command line arguments? Or perhaps an e.g. `.rclipskip`, `.rclipexclude`, `.rclipignore` file that contains directory patterns?

**Comment by groovenectar** [comment]
Specifically in my case there is a problematic `.thumbnails` directory... I don't know that I'd want to hardcode that particular pattern in there though

**Comment by yurijmikhalevich** [comment]
@groovenectar, the feature is in the plans. Thank you for creating a GHI for it!

**Comment by yurijmikhalevich** [comment]
```
  --exclude-dir EXCLUDE_DIR
                        dir to exclude from search, can be specified multiple times; adding this argument
                        overrides the default of ("@eaDir", "node_modules", ".git"); WARNING: the default will be
                        removed in v2
```

The argument was added in https://github.com/yurijmikhalevich/rclip/commit/aa95a0fe9c899ba9a9a7d7b32d44b34ddfe07a90 and released in [v1.1.0](https://github.com/yurijmikhalevich/rclip/releases).
